Template/README.md

```diff
@@ -1 +1,18 @@
-# {PROJECT}
\ No newline at end of file
+# {PROJECT}
+
+### Installation
+
+**CocoaPods:**
+Add the line `pod "{PRODUCT}"` to your `Podfile`
+
+**Carthage:**
+
+Add the line `github "yourusername/{PRODUCT}"` to your `Cartfile`
+
+**Manual:**
+
+Clone the repo and drag the file `{PRODUCT}.swift` into your Xcode project.
+
+**Swift Package Manager:**
+
+Add the line `.Package(url: "https://github.com/yourusername/{PRODUCT}.git", majorVersion: 2)` to your `Package.swift` file.
```

Comment:
My first question about the URL applies here too. Yup, that would be great. Any more info how I can set the 'majorVersion' to '2' by default? P.s sorry if my questions are noobish, I'm new to the open source "universe" and I'm trying to learn as much as I can.